code/game/objects/structures/mirror.dm

```diff
@@ -1,20 +1,33 @@
 //wip wip wup
 /obj/structure/mirror
 	name = "mirror"
-	desc = "Mirror mirror on the wall, who's the most robust of them all?"
+	desc = "It's you!"
 	icon = 'icons/obj/watercloset.dmi'
 	icon_state = "mirror"
 	density = FALSE
 	anchored = TRUE
 	max_integrity = 200
 	integrity_failure = 0.5
+	var/creation_time
 
 MAPPING_DIRECTIONAL_HELPERS(/obj/structure/mirror, 28)
 
 /obj/structure/mirror/Initialize(mapload)
 	. = ..()
 	if(icon_state == "mirror_broke" && !broken)
 		atom_break(null, mapload)
+	creation_time = world.time
+
+/obj/structure/mirror/examine(mob/user)
+	. = ..()
+	if(desc != "It's you!") //I'm really not a fan of hardcoding this, but I don't see another way to do this
```

Comment:
```suggestion
	if(desc != initial(desc))
```

Reply:
I don't want this to apply to the mirror subtypes, though.

Reply:
You should, one is just for directionals which you'd want to keep, and the other is `magic`.

If you only don't want it on `magic`, you should move all of this out into its own proc, then override that proc in the magic subtype, or disable it through a var.